cypress/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```

Comment:
I think we can use a more modern target? `es2020` should work, we can depend on node v16 if needed